plugin/storage/es/mappings/gen_assets.go

```diff
@@ -213,7 +213,7 @@ var _escData = map[string]*_escFile{
 		name:    ".nocover",
 		local:   "plugin/storage/es/mappings/.nocover",
 		size:    43,
-		modtime: 1593482248,
+		modtime: 1595973361,
```

Comment:
no need to change this file

Reply:
Agreed, these unrelated build components were purged but I I think I reintroduced them accidentally in a fixup. They are not, and would not be expected to me in the final submit.